### PR TITLE
Default Pack - small playbook fix and readme update

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6475

## Description
- Fixed an issue where the playbook did not set an initial value for the `Selected Indicators` field, which would result in the field not showing in the layout by default, which could confuse users that want to use the field to select indicators for tagging.
- Updated the pack README to feature the XSOAR 8.2+ content of the Default pack.
- Fix the reference link in the pack README to point to the Default article.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/DefaultPlaybook/README.md [hunk 1]
```diff
@@ -1,14 +1,26 @@
 Sometimes you may have an incident, or just some data at hand, that does not fall into any specific category. When you simply create an incident from scratch or just want to get more information about an indicator without performing a full investigation, we recommend using our Default playbook.
 
-This pack provides you with the Default playbook which automates the much-needed core steps of enrichment and severity calculation for any kind of incident.
+This pack provides you with the Default playbook and layout, which help automate the core steps of every investigation.
+The Default playbook automatically enriches indicators, parses files, detonates files and URLs, maps affected endpoints and calculates a severity for the incident.
+The Default layout is a good match for any incident type, as it dynamially displays mapped or unmapped incident data, files attachments and assets and indicators.
```
Comment (ShirleyDenkberg) [inline Packs/DefaultPlaybook/README.md:5]:
```suggestion
The Default layout is a good match for any incident type, as it dynamically displays mapped or unmapped incident data, file attachments and assets and indicators.

```

File: Packs/DefaultPlaybook/README.md [hunk 2]
```diff
@@ -1,14 +1,26 @@
 Sometimes you may have an incident, or just some data at hand, that does not fall into any specific category. When you simply create an incident from scratch or just want to get more information about an indicator without performing a full investigation, we recommend using our Default playbook.
 
-This pack provides you with the Default playbook which automates the much-needed core steps of enrichment and severity calculation for any kind of incident.
+This pack provides you with the Default playbook and layout, which help automate the core steps of every investigation.
```
Comment (ShirleyDenkberg) [inline Packs/DefaultPlaybook/README.md:3]:
```suggestion
This pack provides you with the Default playbook and layout, which helps automate the core steps of every investigation.

```

**Comment by ShirleyDenkberg** [comment]
@michalgold @tomer-pan @AdiPeret Doc review completed.

**Review by AdiPeret — APPROVED** [review]
Following our session, changes looks good

**Pull request merged**